docs/src/operations/upgrade.md

```diff
@@ -0,0 +1,152 @@
+1. Replace the `/bin` and `/lib` folders with the versions of the new distribution
+1. Start up the broker
+
+### Verifying the Upgrade
+
+The upgrade is successful if the following conditions are met:
+
+* the broker is ready (see [Ready Check](/operations/health.md#ready-check))
+* the broker is healthy (see [Health Check](/operations/health.md#health-check))
+* all partitions are healthy (see the [Metric](/operations/metrics.md#metrics-related-to-health) `zeebe_health`)
+* the stream processors of the partition leaders are in the phase `PROCESSING` (see [Partitions Admin Endpoint](#partitions-admin-endpoint))
+
+If the upgrade failed because of a known issue then a partition change its status to unhealthy, and the log output may contain the following error message:
+
+<details>
+  <summary>Sample Upgrade Error Message</summary>
+  <p>
+
+```
+Unexpected error on recovery happens.
+io.zeebe.engine.processor.InconsistentReprocessingException: Reprocessing issue detected!
+  Restore the data from a backup and follow the recommended upgrade procedure. [cause:
+  "The key of the record on the log stream doesn't match to the record from reprocessing.",
+  log-stream-record: {"partitionId":1,"value":{"version":1,"bpmnProcessId":"parallel-tasks",
+  "workflowKey":2251799813685249,"parentElementInstanceKey":-1,"parentWorkflowInstanceKey":-1,
+  "bpmnElementType":"PARALLEL_GATEWAY","flowScopeKey":2251799813685251,
+  "elementId":"ExclusiveGateway_0tkgnd5","workflowInstanceKey":2251799813685251},
+  "key":2251799813685256,"sourceRecordPosition":4294997784,"valueType":"WORKFLOW_INSTANCE",
+  "timestamp":1601025180728,"recordType":"EVENT","intent":"ELEMENT_ACTIVATING",
+  "rejectionType":"NULL_VAL","rejectionReason":"","position":4294998112},
+  reprocessing-record: {key=2251799813685255, sourceRecordPosition=4294997784,
+  intent=WorkflowInstanceIntent:ELEMENT_ACTIVATING, recordType=EVENT}]
+```
+
+  </p>
+</details>
+
+In this case, the broker should be rolled back to the previous version and the backup should be restored. Ensure that the upgrade was prepared correctly. If it is still unclear why it was not successful then please contact the Zeebe team and ask for guidance.
+
+
+## Rolling Upgrade
+
+Zeebe is designed to allow a rolling upgrade of a cluster. The brokers can be upgrade one after each other. The other brokers in the cluster continue processing until the whole upgrade is done.
+
+1. Upgrade the first broker and wait until it is ready again
+1. Continue with the next broker until all brokers are upgraded
+1. Upgrade the standalone gateways
+
+> Because of current issue, it is not recommended performing a rolling upgrade. Please follow the upgrade procedure instead.
+
+## Partitions Admin Endpoint
+
+This endpoint allows querying the status of the partitions and performing operations to prepare an upgrade.
+
+It is available under `http://{zeebe-broker}:{zeebe.broker.network.monitoringApi.port}/actuator/partitions` (default port: `9600`).
```

Comment:
In 0.23.7 the endpoint is `http://{zeebe-broker}:{zeebe.broker.network.monitoringApi.port}/partitions` and cannot be disabled.